CarDealer/Models/CarInventory.cs

```diff
@@ -66,5 +66,11 @@ public void SetOptionsLevel()
         {
             FinalPrice = InitialPrice;
         }
+
+        public CarInventory Clone()
+        {
+            CarInventory carInventory = (CarInventory)this.MemberwiseClone();
```

Comment:
Aici apelezi metoda DeepCopy 
return (CarInventory) ObjectHelper.DeepCopy < CarInventory > (this);